src/Products/Annotation/Annotator/PointAnnotator.cs

```diff
@@ -1,62 +1,59 @@
-﻿
-using GroupDocs.Annotation.Domain;
-using GroupDocs.Annotation.Domain.Containers;
+﻿using GroupDocs.Annotation.Models;
+using GroupDocs.Annotation.Models.AnnotationModels;
 using GroupDocs.Annotation.MVC.Products.Annotation.Entity.Web;
+using GroupDocs.Annotation.Options;
 using System;
 
 namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
 {
     public class PointAnnotator : BaseAnnotator
     {
-        public PointAnnotator(AnnotationDataEntity annotationData, PageData pageData)
-            : base(annotationData, pageData)
-        {
-        }
-        
-        public override AnnotationInfo AnnotateWord()
+        private PointAnnotation pointAnnotation;
+
+        public PointAnnotator(AnnotationDataEntity annotationData, PageInfo pageInfo)
+            : base(annotationData, pageInfo)
         {
-            return InitAnnotationInfo();
+            pointAnnotation = new PointAnnotation()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372470)